game/hiscore.cc

```diff
@@ -39,7 +43,7 @@ Hiscore::HiscoreVector Hiscore::queryHiscore(unsigned max, unsigned playerid, un
 }
 
 bool Hiscore::hasHiscore(unsigned songid) const {
-	for (auto const& h: m_hiscore) if (songid == h.songid) return true;
+	for (auto const& h: m_hiscore) if (songid == h.songid && currentLevel() == h.level) return true;
```

Comment:
`currentLevel()` is an int while `h.level` is unsigned.
The current pipelines are nagging about this comparison:
`e/hiscore.cc:37:22:parison of integer expressions of different signedness: ‘pare`